turbine/src/broadcast_stage/standard_broadcast_run.rs

```diff
@@ -25,7 +30,9 @@ use {
 pub struct StandardBroadcastRun {
     slot: Slot,
     parent: Slot,
+    parent_block_id: Hash,
     chained_merkle_root: Hash,
+    double_merkle_leaves: Vec<Hash>,
```

Comment:
Are we going to have a special type for this (not Hash)? I seem to recall a PR from Akhi with a Slice Hash or something